components/02-molecules/accordion/accordion.twig

```diff
@@ -0,0 +1,37 @@
+{% set accordion__base_class = 'accordion' %}
+
+<div {{ bem(accordion__base_class) }} data-component-width="content">
+  <div {{ bem('inner', [], accordion__base_class) }}>
+    {% include "@atoms/typography/headings/heading.twig" with {
+      heading__level: '3',
+      heading__blockname: accordion__base_class,
+      heading: accordion__heading,
+    } %}
+    {% embed "@atoms/lists/list.twig" with {
+      list__base_class: 'controls',
+      list__blockname: accordion__base_class,
+      list__attributes: {'aria-labels': 'Section controls'},
+    } %}
+      {% block list__content %}
+        {% include "@atoms/buttons/button.twig" with {
+          button__content: 'Expand all',
+          button__base_class: 'toggle-all',
+          button__modifiers: ['expand'],
+          button__blockname: accordion__base_class,
+        } %}
+        {% include "@atoms/buttons/button.twig" with {
+          button__content: 'Collapse all',
+          button__base_class: 'toggle-all',
+          button__modifiers: ['collapse'],
+          button__blockname: accordion__base_class,
```

Comment:
Both collapse and expand buttons need aria-expanded/collapsed to be announced. Also, can we combine the interactions into one button (like the Figma design)? 

<img width="754" alt="Screen Shot 2022-03-22 at 8 11 15 PM" src="https://user-images.githubusercontent.com/12991593/159597118-13ef3ffa-3bd4-4095-b169-e7e8ca9faad6.png">

Reply:
I just got off a call with Mike V and we chatted about the Expand/Collapse toggle pattern here. We are thinking that a true toggle button will be more clear to sighted and blind users. There are two states that can be turned on or off and the two buttons || one button with two labels does not convey this as clearly as a toggle would. 

Reading about accessible toggles: [https://adrianroselli.com/2019/08/under-engineered-toggles-too.html](https://adrianroselli.com/2019/08/under-engineered-toggles-too.html)